### ISSUE TITLE
Error when building with make on Windows 10

### ISSUE THREAD

**Issue by gruossomonica** [homuler/MediaPipeUnityPlugin#68]
Hi,
I installed the dependencies and run the MediaPipe HelloWorld! example without problems, but an error occurred during the building process with make (step 4 of the [Installation section](https://github.com/homuler/MediaPipeUnityPlugin#installation) of your readme).

Windows 10
OpenCV 3.4.10 (since it was installed under F:\opencv, I changed the `C/WORKSPACE` file accordingly)
.NET Core SDK 3.1 and .NET Core runtime 2.1
Bazel 4.0
NumPy 1.20.1
Python 3.7.0 (accessible using py -3 since python 2.7 is also installed on my computer)
MSYS2, Visual C++ Build Tools 2019, and WinSDK were installed following the [MediaPipe installation guide](https://google.github.io/mediapipe/getting_started/install.html#installing-on-windows).

```
F:\MediaPipeUnityPlugin>make cpu PYTHON_BIN_PATH="C://Python37//python.exe"
cd C && bazel --output_user_root=C:/_bzl build -c opt --define MEDIAPIPE_DISABLE_GPU=1 --action_env PYTHON_BIN_PATH="C://Python37//python.exe" mediapipe_api:mediapipe_desktop mediapipe_api:mediapipe_assets mediapipe_api:mediapipe_proto_srcs
DEBUG: C:/_bzl/fsfojsgo/external/rules_foreign_cc/workspace_definitions.bzl:15:10: WARNING: This branch is deprecated and no longer recieving updates. Please update to `main` or choose a specific commit to pin in your workspace.
INFO: Analyzed 3 targets (1 packages loaded, 2418 targets configured).
INFO: Found 3 targets...
INFO: From Compiling mediapipe/framework/deps/ret_check.cc [for host]:
C:\_bzl\fsfojsgo\execroot\mediapipe_api\bazel-out\host\bin\external\com_github_glog_glog\_virtual_includes\windows_glog_headers\glog\logging.h(1660) : warning C4722: 'google::NullStreamFatal::~NullStreamFatal': il distruttore non restituisce mai un valore, potenziale perdita di memoria
INFO: From Compiling mediapipe/framework/deps/status.cc [for host]:
C:\_bzl\fsfojsgo\execroot\mediapipe_api\bazel-out\host\bin\external\com_github_glog_glog\_virtual_includes\windows_glog_headers\glog\logging.h(1660) : warning C4722: 'google::NullStreamFatal::~NullStreamFatal': il distruttore non restituisce mai un valore, potenziale perdita di memoria
INFO: From Compiling mediapipe/framework/deps/status_builder.cc [for host]:
C:\_bzl\fsfojsgo\execroot\mediapipe_api\bazel-out\host\bin\external\com_github_glog_glog\_virtual_includes\windows_glog_headers\glog\logging.h(1660) : warning C4722: 'google::NullStreamFatal::~NullStreamFatal': il distruttore non restituisce mai un valore, potenziale perdita di memoria
INFO: From Compiling absl/debugging/internal/address_is_readable.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/debugging/internal/elf_mem_image.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/flags/internal/commandlineflag.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/flags/flag.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/debugging/internal/vdso_support.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/flags/internal/program_name.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/base/internal/low_level_alloc.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/strings/cord.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/types/bad_optional_access.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/flags/commandlineflag.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
INFO: From Compiling absl/debugging/symbolize.cc:
cl : warning della riga di comando D9025 : override di '/w' con '/W3'
ERROR: C:/_bzl/fsfojsgo/external/org_tensorflow/tensorflow/core/util/BUILD:367:24: Action external/org_tensorflow/tensorflow/core/util/version_info.cc failed: (Exit 1): bash.exe failed: error executing command C:/msys64/usr/bin/bash.exe -c bazel-out/x64_windows-opt-ST-e0f78fafe98f/bin/external/org_tensorflow/tensorflow/tools/git/gen_git_source.exe --generate "$@" --git_tag_override=${GIT_TAG_OVERRIDE:-}  ... (remaining 4 argument(s) skipped)
Traceback (most recent call last):
  File "c:\users\monica~1\appdata\local\temp\Bazel.runfiles_3eur5w\runfiles\org_tensorflow\tensorflow\tools\git\gen_git_source.py", line 32, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins
INFO: Elapsed time: 163.374s, Critical Path: 146.69s
INFO: 453 processes: 129 internal, 324 local.
FAILED: Build did NOT complete successfully
make: *** [Makefile:46: cpu] Error 1
```

How can be solved?
I look forward to your reply.

Best Regards,
Monica

**Comment by jsaon-L** [comment]
try use bazel3.7

**Comment by homuler** [comment]
> ImportError: No module named builtins

It looks like `builtins` module is missing.
Will you try this?
```sh
pip install future
```

**Comment by gruossomonica** [comment]
Hi, 
thanks for your support. I installed `future` and used Bazel 3.5.0, as indicated.
Now the error is the following
```
ERROR: Analysis of target '//mediapipe_api:mediapipe_desktop' failed; build aborted: Either the path attribute of android_sdk_repository or the ANDROID_HOME environment variable must be set.
```
Currently, the same error appears running the MediaPipe HelloWorld! example. I solved it with Bazel 3.4.0 (I found useful info [here](https://github.com/google/mediapipe/issues/1094)) and then I was able to run the MediaPipe HelloWorld! example without problems.

Finally, `future` and Bazel 3.4.0 solved all problems and now it works.
